frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.scss

```
@@ -69,32 +80,45 @@ $gutter: 1.8em;
     }
     &.is-text-center {
       text-align: center;
+      max-width: 100px;
+      margin: 0 var(--pf-global--spacer--md);
```

With the pills being `120px` wide and the `--pf-global-spacer--md` being `16px`, the `max-width` is never reached (120 - 16 * 2 = 88) unless the variable is changed.

Are we just looking to avoid touching the sides? I think we can achieve this with:
```suggestion
      margin: 0 auto;
```
While keeping the `max-width` to prevent it from squishing against the sides.

Is there a style guideline that says we should be using this variable to space things?

---

Also, @serenamarie125 the Visual Spec document on the ticket states the text should be 75px wide - is that only when an icon is in play?